{
  try {
    assertCanExecute();
  }
 catch (  AssertionError e) {
    return false;
  }
  return true;
}
